### PR TITLE
[golang] Fix things commented out in 1.19rc2

### PR DESCRIPTION
**Pull request by dims**
Signed-off-by: Davanum Srinivas <davanum@gmail.com>

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by aojea** [comment]
/retest

leetve failures

**Comment by aojea** [comment]
lol leetve job

> Detected go version: go version go1.18.3 linux/amd64.
> Kubernetes requires go1.19 or greater.
> Please install go1.19 or later.}

**Comment by dims** [comment]
LOL indeed :) BUT we still don't have kubekins-e2e with 1.19 GA yet. so this has to wait a bit more.

**Comment by dims** [comment]
thanks @liggitt !

**Review by cpanato — APPROVED** [review]
/lgtm

**Review by palnabarun — APPROVED** [review]
/lgtm

File: cmd/kube-scheduler/app/options/configfile.go [hunk 1]
```diff
@@ -53,7 +53,7 @@ func loadConfig(data []byte) (*config.KubeSchedulerConfiguration, error) {
 		// more details.
 		cfgObj.TypeMeta.APIVersion = gvk.GroupVersion().String()
 		if cfgObj.TypeMeta.APIVersion == configv1beta2.SchemeGroupVersion.String() {
-			klog.Warning("KubeSchedulerConfiguration v1beta2 is deprecated in v1.25, will be removed in v1.26")
+			klog.V(0).InfoS("KubeSchedulerConfiguration v1beta2 is deprecated in v1.25, will be removed in v1.26")
```
Comment (liggitt) [inline cmd/kube-scheduler/app/options/configfile.go:56]: per https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md#change-log-functions, klog.Warning should just be klog.InfoS, I think

Comment (dims) [reply cmd/kube-scheduler/app/options/configfile.go:56]: ah fixing

Comment (dims) [reply cmd/kube-scheduler/app/options/configfile.go:56]: i saw this pattern of V(0) else where in the code base ...

**Review by palnabarun — APPROVED** [review]
/lgtm

File: cmd/kube-scheduler/app/options/configfile.go (hunk 1 above)
Comment (pohly) [reply cmd/kube-scheduler/app/options/configfile.go]: Must have slipped through code review. It doesn't make much difference, `klog.V(0).InfoS` and `klog.InfoS` are semantically equivalent, the former just has a bit more overhead.

**Comment by dims** [comment]
also see https://github.com/kubernetes/kubernetes/issues/111297

**Pull request merged**